dotcom-rendering/docs/snyk/how-to.md

```diff
@@ -0,0 +1,47 @@
+## Snyk how to guide
+
+### Pre-requisite
+Ask your EM or DevX to add you to `guardian-dotcom-n2y` Snyk organisation.
+
+### How to log in
```

Comment:
This isn't a necessary first step as  the `snyk auth` command will drive you to this  flow anyhow, I wonder if this just makes the docs longer to read.